connector_search_engine/models/se_binding.py

```diff
@@ -86,7 +86,8 @@ def _jobify_recompute_json(self, force_export=False):
         description = _(
             "Recompute %s json and check if need update" % self._name
         )
-        for record in self:
+        # The job creation with tracking is very costly. So disable it.
+        for record in self.with_context(tracking_disable=True):
```

Comment:
hmm I wonder where tracking happens? :thinking: 
AFAIS this should be put into `recompute_json` on the write call only `record.with_context(tracking_disable=True)..write(vals)`.
Not to mention that is weird that tracking does something IF the fields are not tracked.
Which field is tracked?